### PR TITLE
Safe mode

### PR DESCRIPTION
**Pull request by mkhab7**
Some changes and refactoring on safe mode

### CODE REVIEW

File: src/RunningMode/Webhook.php [hunk 1]
```diff
@@ -61,18 +61,28 @@ public function processUpdates(Nutgram $bot): void
...
     }
 
+
+    /**
+     * @param $ip
+     * @return float
+     */
+    private function ipToLong($ip): float
+    {
+        return (float)sprintf("%u", ip2long($ip));
```
Comment (SergiX44) [inline src/RunningMode/Webhook.php:83]: Why a long -> to string -> to float is done?

Comment (SergiX44) [reply src/RunningMode/Webhook.php:83]: To me looks totally unecessary, also the name of the function should be ipToFloat, given the return value
https://3v4l.org/iIngp#v8.0.15

Comment (mkhab7) [reply src/RunningMode/Webhook.php]: Yes ,actually this method is not necessary

**Comment by SergiX44** [comment]
Can you please ad a unit test for it?

**Pull request merged**